README.md

```diff
@@ -35,3 +35,18 @@ kubectl nodepools nodes $nodepool
 ```
 
 Where `$nodepool` should be the name of an existing node pool.
+
+### Using a custom node pool label
+Using a custom label will use that label over the cloud ones.
+
+```shell
+  -l, --label string                   Label to group nodes into pools with
+```
```

Comment:
Instead of having to keep the label help in sync between the code and README, how about something like this?

```suggestion
If your cluster uses a different label than the ones supported in code, you can pass a custom label using the `--label/-l` flag as in the following examples:
```

Reply:
Updated.